datacenter/dtr/2.1/guides/release-notes.md

```diff
@@ -13,6 +13,26 @@ known issues for each DTR version.
 You can then use [the upgrade instructions](install/upgrade.md),
 to upgrade your installation to the latest release.
 
+## DTR 2.1.5
+
+(16 March 2017)
+
+**Bug fixes**
+
+* Fix memory leaks in DTR API and Registry
+* Fixed an issue causing RethinkDB to not start due to DNS errors when
+the RethinkDB containers were not restarted at the same time
+* When `--dtr-external-url` was set with uppercase letters, users would not be
+able to log into DTR. This is now fixed
+* The Users page was always displaying a user creation form. now this only shows
+after clicking the 'New user' button
+* The UI seemed to load indefinitely when viewing a manifest without a size field.
+Now if a manifest doesn't a a size, nothing is displayed to the user
+
+**General improvements**
+
+* Made the DTR installer more resilient when using an NFS storage backend
```

Comment:
Also, this should really mention that it fixes a problem when performing upgrades and reconfiguring if the user is using NFS.  This should be medium severity.